recipe/meta.yaml

```diff
@@ -9,20 +9,20 @@ package:
 
 source:
   url: https://pypi.io/packages/source/{{ pypi_name[0] }}/{{ pypi_name }}/{{ pypi_name }}-{{ version }}.tar.gz
-  sha256: f970995ec2bb815e2fdaf7977b26b2091e1e386f0f42eafd5ac811953dc5d445
+  sha256: f3bd36cd261b440a88a1190b1becca0578fee90b4b62decc796932fdd5ae8839
 
 build:
-  skip: True  # [py<37]
+  skip: True  # [py<38]
```

Comment:
I would keep py<37 as this is the compatibility set in the upstream. We will implicitly skip building for py37 as this is specified in aggregate cbc.

Reply:
I thought we were no longer supporting Python 3.7?

Reply:
Ok I'll make that fix for now. Thanks!